src/ffi.rs

```diff
@@ -355,7 +355,7 @@ pub fn public_keys(args: i32, len: i32) -> i64 {
         None => return utils::fail(),
     };
 
-    let keys = (0..=MAX_KEY)
+    let keys = (0..MAX_KEY)
         .map(|idx| key::derive_pk(&seed, idx as u64))
```

Comment:
Same here. Just to be sure, check if we have any part of the code where this pattern is repeated, and replace it with the array outside the loop.